store/tikv/2pc.go

```diff
@@ -676,6 +676,7 @@ func (action actionPessimisticLock) handleSingleBatch(c *twoPhaseCommitter, bo *
 		IsFirstLock:  c.isFirstLock,
 		WaitTimeout:  action.lockWaitTime,
 	}, pb.Context{Priority: c.priority, SyncLog: c.syncLog})
+	lockWaitStartTime := time.Now()
 	for {
```

Comment:
We need to update the `action.lockWaitTime`. so if the wait timeout is less than 3s，we can return sooner.

Reply:
roger